src/positions/slice.ts

```diff
@@ -1,22 +1,25 @@
 import { createSlice, PayloadAction } from '@reduxjs/toolkit'
 import { REHYDRATE, RehydrateAction } from 'redux-persist'
 import { getRehydratePayload } from 'src/redux/persist-helper'
-import { Position, Shortcut, ShortcutStatus } from './types'
 import { NetworkId } from 'src/transactions/types'
+import { SerializableTransactionRequest } from 'src/viem/preparedTransactionSerialization'
+import { Address, Hex } from 'viem'
+import { Position, Shortcut, ShortcutStatus } from './types'
 
 type Status = 'idle' | 'loading' | 'success' | 'error'
 
-export type TriggeredShortcuts = Record<
+type TriggeredShortcuts = Record<
   string,
-  {
-    status: ShortcutStatus
-    transactions: RawShortcutTransaction[]
-    appName: string
-    appImage: string
-    appId: string
-    networkId: NetworkId
-    shortcutId: string
-  }
+  | {
+      status: ShortcutStatus
+      transactions: RawShortcutTransaction[]
+      appName: string
+      appImage: string
+      appId: string
+      networkId: NetworkId
+      shortcutId: string
+    }
+  | undefined
```

Comment:
when can this be `undefined`?

Reply:
If we pass an ID that wasn't stored.
More to be on the safe side, to force callers to check for that case.